src/new_game.c

```diff
@@ -170,7 +164,7 @@ void NewGameInitData(void)
     ClearSav1();
     ClearMailData();
     gSaveBlock2Ptr->specialSaveWarpFlags = 0;
-    gSaveBlock2Ptr->field_A8 = 0;
+    gSaveBlock2Ptr->unusedFlagField = 0;
```

Comment:
Should be left as-is until we've exhausted all gSaveBlock2Ptr references from the assembly